opencog/guile/CMakeLists.txt

```diff
@@ -36,6 +37,7 @@ TARGET_LINK_LIBRARIES(logger
 	smob
 	${GUILE_LIBRARIES}
 	${COGUTIL_LIBRARY}
+    ruleengine # for the ure logger
```

Comment:
The reason I need that is to be able to recognize a Logger* type, with COG_LOGGER. I'd be more than happy to move that in the module if there's a way.